packages/appstore/src/components/transaction-list/completed-transaction/completed-transaction.tsx

```diff
@@ -3,12 +3,13 @@ import { AppLinkedWithWalletIcon, Text, WalletIcon } from '@deriv/components';
 import { getTradingAccountName, getWalletCurrencyIcon } from '@deriv/utils';
```

Comment:
Suppose we need to add tests for the component

Reply:
Added a test file